firedantic/_sync/model.py

```diff
@@ -96,23 +96,44 @@ def get_document_id(self):
             self._validate_document_id(doc_id)
         return getattr(self, self.__document_id__, None)
 
+    _OrderDirection = Union[Literal["ASCENDING"], Literal["DESCENDING"]]
+    _OrderBy = List[tuple[str, _OrderDirection]]
```

Comment:
https://docs.python.org/3.8/library/typing.html#annotating-tuples

If we intend to support Python 3.8 you need to use `typing.Tuple`